README.rst

```diff
@@ -242,6 +242,43 @@ Run qa tests:
 
     ./run-qa-checks
 
+Quickstart Guide
+----------------
+
+Creating a topology
+^^^^^^^^^^^^^^^^^^^
+
+1. Create a topology object by going to *Network Topology* > *Topologies*
+   > *Add topology*.
+2. Give an appropriate label to the topology.
+3. Select the *topology format* from the dropdown menu. The *topology format*
+   determines which parser should be used to process topology data.
+4. Select the *Strategy* for updating this topology.
+
+   - If you are using `FETCH strategy <#fetch-strategy>`_, then enter the
+     URL for fetching topology data in the *Url* field.
+   - If you are using `RECEIVE strategy <#receive-strategy>`_, you will get the
+     *URL* for sending topology data. The *RECEIVE* strategy provides an
+     additional field *expiration time*. This can be used to add delay in
+     marking missing links as down.
+
+.. image:: https://github.com/openwisp/openwisp-network-topology/raw/docs/docs/quickstart-topology.gif
+
+Sending data for topology with RECEIVE strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Copy the *URL* generated by OpenWISP for sending the topology data.
+2. Send the topology data using ``POST`` method as follows:
+
+.. code-block:: shell
+
+    curl -s -X POST \
+        --data-binary @openvpn-status.log \
```

Comment:
please use an example in which the data is taken with a command, because not all the networking software we use have log files, many programs show the output with a command.

Please also specify that in this example we are sending the status log data from OpenVPN but the same code can be applied to other formats.